README.md

```diff
@@ -35,7 +35,8 @@ Our code of conduct is [Contributor Covenant](CODE_OF_CONDUCT.md). All contribut
 
 ## Maintainers
 
-vacat
+- Andrew Weaver (@privatezero)
+- Reto Kromer (@retokromer)
 
```

Comment:
Dave Rice (@dericed) should be added to this section as well

Reply:
For sure, if he agrees.